docs/integrations/visual-context-system/scripts/statusline.sh

```
@@ -0,0 +1,164 @@
+if [ -f "$AIOS_STATE_FILE" ]; then
+  DISPLAY_TITLE=$(jq -r '.project.displayTitle // ""' "$AIOS_STATE_FILE" 2>/dev/null)
+  TITLE_EMOJI=$(jq -r '.project.titleEmoji // ""' "$AIOS_STATE_FILE" 2>/dev/null)
+  PROJECT_EMOJI=$(jq -r '.project.emoji // ""' "$AIOS_STATE_FILE" 2>/dev/null)
+  PROJECT_NAME=$(jq -r '.project.name // ""' "$AIOS_STATE_FILE" 2>/dev/null)
+  PROGRESS=$(jq -r '.status.progress // ""' "$AIOS_STATE_FILE" 2>/dev/null)
+  STATUS_EMOJI=$(jq -r '.status.emoji // ""' "$AIOS_STATE_FILE" 2>/dev/null)
+
+  # Build context (truncate if needed)
+  if [ -n "$DISPLAY_TITLE" ] && [ "$DISPLAY_TITLE" != "null" ]; then
+    # Prepend titleEmoji if available
+    local emoji_prefix=""
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)